feature/stamps/src/main/java/io/github/droidkaigi/confsched2023/stamps/section/StampList.kt

```diff
@@ -33,6 +33,7 @@ private const val DoubleItemSpanCount = 2 / 2
 @Composable
 fun StampList(
     stamps: ImmutableList<Stamp>,
+    detailDescription: String,
     @androidx.annotation.RawRes
     stampLottieRawId: Int?,
```

Comment:
In our policy, we want to have UiState for sections.
But we might not have time. So I'll leave it to you
https://github.com/DroidKaigi/conference-app-2023#section